### ISSUE TITLE
Лендинг

### ISSUE THREAD

**Issue by nicothin** [nicothin/nesia#26]
@vladimir075 @nadezdafrolova 

Макеты [настольные](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3041%3A1491) и [мобильные](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3161%3A1701)

Задача: сверстать.

**Comment by vladimir075** [comment]
верно

**Comment by nicothin** [comment]
https://nicothin.pro/nesia/landing.html

**Comment by vladimir075** [comment]
@nicothin 
Николай, Олег начал внедрять, там пропустили в дизайне в форме регистрации две вещи 1) ошибочные состояния полей красные 2) Перед полем телефона в той же строке, но отдельным полем нужно поле для кода страны (при выборе страны туда будет вставляться код телефона). Если можно, прошу сделать это оперативно.
@greeceruit

**Comment by nicothin** [comment]
Сделал на уровне вёрстки (функциональность заполнения поля с кодом страны сделает Олег?)

Для ошибочных состояний дал 2 примера, оба подразумевают добавление класса (бутстраповский `has-error` или мой `b-form-group--error`)

**Comment by vladimir075** [comment]
1) Можно ли добавить надписи под ошибочными состояниями? Например email is not valid, the field is obligatory итд
2) можно ли поле кода чуть пошире. Встречаются до семизначных, с учетом знака плюс получается восемь цифр ))

**Comment by vladimir075** [comment]
@greeceruit Олег, функциональность ведь ты делаешь?

**Comment by nicothin** [comment]
> 1. Можно ли добавить надписи под ошибочными состояниями? Например email is not valid, the field is obligatory итд

Да, такая возможность там есть, добавил в демку текста с описанием ошибки (это не новое в вёрстке, оно уже было ранее).

> 2. можно ли поле кода чуть пошире. Встречаются до семизначных, с учетом знака плюс получается восемь знаков ))

Я думал об этом, поэтому ширина поля ввода кода страны указана в самой разметке. Возможно, грамотнее будет определять ее динамически, исходя из кол-ва символов в самом коде страны.
Но если это заморочно, монжо в любой момент вписать другую ширину без к-л заморочек с компиляцией стилей:

![image](https://user-images.githubusercontent.com/3499680/182185527-57f39705-3220-4750-b160-5a22a1b509dc.png)

**Comment by vladimir075** [comment]
Добро, спасибо. Олег, прошу принять в разработку @greeceruit

**Comment by greeceruit** [comment]
> Сделал на уровне вёрстки (функциональность заполнения поля с кодом страны сделает Олег?)
> 

Да, сделаю....

> Для ошибочных состояний дал 2 примера, оба подразумевают добавление класса (бутстраповский `has-error` или мой `b-form-group--error`)
>

Понятно....
